doc/releases/changelog-dev.md

```diff
@@ -4,7 +4,53 @@
 
 <h3>New features since last release</h3>
 
-* `DefaultQubit` devices now natively support parameter broadcasting.
+* The gradient transform `qml.gradients.param_shift` now accepts the new Boolean keyword
+  argument `broadcast`. If it is set to `True`, broadcasting is used to compute the derivative.
+  [(#2749)](https://github.com/PennyLaneAI/pennylane/pull/2749)
+
+  For example, for the circuit
+
+  ```python
+  x, y = np.array([0.4, 0.23], requires_grad=True)
+
+  with qml.tape.QuantumTape() as tape:
+      qml.RX(x, wires=0)
+      qml.CRY(y, wires=[0, 1])
+      qml.expval(qml.PauliZ(0) @ qml.PauliZ(1))
+  ```
+
+  we may compute the derivative via
+
+  ```pycon
+  >>> dev = qml.device("default.qubit", wires=2)
+  >>> tapes, fn = qml.gradients.param_shift(tape, broadcast=True)
+  >>> len(tapes)
+  2
+  >>> (tapes[0].batch_size, tapes[1].batch_size)
+  (2, 4)
+  ```
+
+  For `broadcast=False` (the default), multiple unbroadcasted tapes are created as before.
+
+  ```pycon
+  >>> tapes, fn = qml.gradients.param_shift(circuit.qtape, broadcast=False)
+  >>> len(tapes)
+  6
+  >>> [t.batch_size for t in tapes]
+  [None, None, None, None, None, None]
+  ```
```

Comment:
Let's move this example to the docstring.

Reply:
You mean everything after "For `broadcast=False`..." ?

Reply:
This is outdated due to the reduction to a pure QNode example.